modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-by-using-the-checlusterrestore-object.adoc

```diff
@@ -0,0 +1,64 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-by-using-the-checlusterrestore-object_{context}"]
+= Restoring a {prod-short} instance from a backup by using the CheClusterRestore custom object
+
+You can restore a {prod-short} instance from a backup snapshot by using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; that is, editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+IMPORTANT: The `CheClusterRestore` custom object is unusable for recovering a {prod-short} instance of an earlier version of {prod-short}. Use `{prod-cli}` to recover a {prod-short} instance of an earlier version of {prod-short}; see xref:restoring-a-che-instance-from-a-backup-by-using-prod-cli_{context}[].
+
+.Prerequisites
+
+* Backup snapshot of a {prod-short} instance on any of the following:
+** xref:backing-up-of-che-instances-to-an-sftp-backup-server.adoc[SFTP backup server]
+** xref:backing-up-of-che-instances-to-amazon-s3.adoc[Amazon S3 or S3 API compatible backup server]
+** xref:backing-up-of-che-instances-to-a-rest-backup-server.adoc[REST backup server]
+** xref:backing-up-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
```

Comment:
Internal backup server is a particular case of REST backup server.

Reply:
No worries, it's stated there.